src/layout/NavigationLink.js

```diff
@@ -7,17 +7,32 @@ import { Link } from 'react-router';
 export class NavigationLink extends React.Component {
   static propTypes = {
     className: PropTypes.string,
-    to: PropTypes.string.isRequired
+    to: PropTypes.string,
+    href: PropTypes.string
+  }
+
+  createElement(type, href, otherProps, className) {
+    return React.createElement(
+      type, Object.assign({}, otherProps, {
+        href: href,
+          className: 'mdl-navigation__link'
+        })
+      )
   }
 
   render() {
-    const { className, to } = this.props;
+    const { className, to, href, ...otherProps } = this.props;
+
     const classes = classNames('mdl-navigation__link', className);
 
+    let element = href
```

Comment:
I think something like the following would be better (_avoiding extra `createElement` in the component_) :

```javascript 
render(){
  const { className, to, href, ...otherProps } = this.props;
  const classes = classNames('mdl-navigation__link', className);

  const tag = to ? Link: 'href';
  // below is a js trick I've already showed to you, i.e.: { to } expands to { to: to }.
  let baseProps =  Object.assign( (to ? { to } : { href }), {className: classes});
  
  return (
    {React.createElement(
      tag, Object.assign({}, otherProps, baseProps)
     )}
  );
}
```

What I'm not sure about is that if we have to import `Link` in this file.